Add TreeView descriptor article (#770)
telerik/maui-docs: controls/treeview/descriptor.md

---

> title: TreeView Descriptor

Reviewer: If I get the idea correctly, the main purpose of the `TreeViewDescriptor` class is to control the hierarchy and where in the hierarchy an item gets displayed. If this is the case, consider a title that is more use-case-centric instead of class-name-centric. If my assumption is wrong, ignore the title-related suggestions in this review.

```suggestion
title: Item Hierarchy
```

> page_title: .NET MAUI TreeView Documentation - TreeView Descriptor

Reviewer: ```suggestion
page_title: .NET MAUI TreeView Documentation - Item Hierarchy
```

description: Learn how to define the appearance of the items in the Telerik UI for .NET MAUI TreeView control.
position: 2
slug: treeview-descriptor
---

> # .NET MAUI TreeView Descriptor

Reviewer: ```suggestion
# .NET MAUI TreeView Item Hierarchy
```

> The Telerik .NET MAUI `TreeViewDescriptor` is part of TreeView items visualization. The `TreeViewDescriptor` defines the data items' hierarchy and how each item is visualized.

Reviewer: ```suggestion
To specify the data items' hierarchy and how each item is visualized, use the `TreeViewDescriptor` class. For more flexibility, you can define multiple descriptors in the TreeView.
```

> The `TreeViewDescriptor` class exposes the following properties described in the table below: 

Reviewer: ```suggestion
The `TreeViewDescriptor` class exposes the following properties: 
```

| Property | Description |
| -------- | ----------- |
| `TargetType` (`System.Type`) | Specifies the type of the data item the descriptor refers to. |
| `IdentityMemberPath` (`string`) | Specifies the path to the identity member of the data item. The identifier member is used to identify the item in the hierarchy. If the property is not specified, the entire item is used instead. |
| `DisplayMemberPath ` (`string`) | Specifies the path to the display member of the data item. The display member is used to visualize the item on the screen. If the property is not specified, the entire item is used instead. |
| `ItemsSourcePath` (`string`) | Specifies the path to the child items source of the data item. |
| `ItemTemplate` (`DataTemplate`) | Specifies the template applied to the respective type of items. |
| `ItemStyle` (`Style` with a target type of `TreeViewItemView`) | Specifies the style of the TreeView items. |

> You can define multiple descriptors in the TreeView.

> Here is an example with TreeView descriptor:

Reviewer: ```suggestion
Here is an example demonstrating how to define the data items' hierarchy by using the TreeView descriptor:
```
